tests/make_repo_test.py

```diff
@@ -256,3 +256,23 @@ def test_docker_image_integration(in_git_dir):
     assert _cmd('git', 'log', '--oneline')
 
     # TODO: test that the package is installable
+
+
+def test_docker_image_integration_official_repo(in_git_dir):
+    make_repo(
+        '.',
+        language='docker_image', name='python', description='',
+        entry='python', id='python-docker', match_key='files',
+        match_val=r'\.py$', args='[]', require_serial='false',
+        minimum_pre_commit_version='0',
+    )
```

Comment:
Not really a valid pre-commit hook, but there are so few in the official repo :shrug:
Also this is a good, but time consuming test for the pagination requests.